### ISSUE TITLE
Overflow on bn_conv1 with TRAIN_BN = False

### ISSUE THREAD

**Issue by benqua** [matterport/Mask_RCNN#573]
Hi,

I train a resnet50 with the config `TRAIN_BN = False` and, sometimes, there is an overflow on the layer bn_conv1. 
In this case, `visualize.display_weight_stats(model)` returns:
```
bn_conv1/moving_mean:0 | (64,) | -173.0470 | +116.3013 | +44.5654
bn_conv1/moving_variance:0*** Overflow? | (64,) | +0.0000 | +146335.3594 | +21847.9668
```

How is it possible to hit overflow on batch normalization layers when TRAIN_BN is set to False?

**Comment by baoruxiao** [comment]
I found that keras's resnet50 pretrain weights already overflow the bn_conv1 weight.

**Comment by genausz** [comment]
I found the problem too.

**Comment by henrique** [comment]
I have just worked out the same. It's also in https://github.com/matterport/Mask_RCNN/blob/master/samples/coco/inspect_weights.ipynb

From the values, I'd guess someone forgot to scale their images prior to training?
Has anyone found a solution for this?
Maybe resetting the values and retraining?

I also don't get why mold_image() doesn't divide the pixel values by their variance. Gradient descent doesn't like those huge values. Or is it done somewhere else?
